### PR TITLE
Added Image Registry and ChaosHub docs

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

**What this PR does / why we need it**:
This PR adds:
- ChaosHub docs
- Image Registry docs

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/concepts/chaoshub.md [hunk 1]
```diff
@@ -4,4 +4,88 @@ title: ChaosHub
 sidebar_label: ChaosHub
 ---
 
-## Coming soon
+---
+
+## Introduction
+ChaosHub allows a user to orchestrate workflows from the Public **[ChaosHub](http://hub.litmuschaos.io/)** or an alternate source for the Experiments (basically, a **[fork](https://github.com/litmuschaos/chaos-charts)** of the public hub with custom experiments).
```
Comment (Saranya-jena) [inline website/docs/concepts/chaoshub.md]: We should use the second person to address, as this convention is used everywhere. You can write - `ChaosHub allows a user` -> `ChaosHub allows a you`

File: website/docs/concepts/chaoshub.md [hunk 2]
```diff
@@ -4,4 +4,88 @@ title: ChaosHub
...
+
+## Prerequisites
+The following should be required before creating a Chaos Workflow:
```
Comment (Saranya-jena) [inline website/docs/concepts/chaoshub.md]: following should -> following are

File: website/docs/concepts/chaoshub.md [hunk 3]
```diff
@@ -4,4 +4,88 @@ title: ChaosHub
...
+- [ChaosCenter](chaos-center)
+- [ChaosAgent](agent)
+- Fork of [Chaos-Charts](https://github.com/litmuschaos/chaos-charts) repository  
+
+:::note
+  An active internet connection is required to clone the git repository for the first time installation.
+:::
+
+## Connecting a Git repository using ChaosHub
+With ChaosHub, a user can construct chaos workflows by selecting, tuning and sequencing different experiments together from their connected ChaosHubs.
```
Comment (Saranya-jena) [inline website/docs/concepts/chaoshub.md]: user -> you

File: website/docs/concepts/chaoshub.md [hunk 4]
```diff
@@ -4,4 +4,88 @@ title: ChaosHub
...
+
+User can make changes in their forked repositories and sync it with the Portal to get the latest changes from the fork.
```
Comment (Saranya-jena) [inline website/docs/concepts/chaoshub.md]: Same here

File: website/docs/concepts/chaoshub.md (hunk 2 above)
Comment (vanshBhatia-A4k9) [inline website/docs/concepts/chaoshub.md]:
```suggestion
The following are the prerequisites for creating a Chaos Workflow:
```

`should` indicates a possibility.. but it is in fact a mandate in this case..

**Review by vanshBhatia-A4k9 — APPROVED** [review]
Left a suggestion, PTAL. Rest is great :rocket:

File: website/docs/user-guides/image-registry.md [hunk 5]
```diff
@@ -3,5 +3,54 @@ id: image-registry
...
+---
+
+## Image Registry
+A container image registry can be defined as a collection of repositories that store container image.
+These can be either public or private. Few of the container image registries are Docker, Red Hat Quay, Google Container Registry. 
+By default LitmusChaos uses DockerHub for managing the different images. These images are then used in Chaos Workflows. Few images that are used in the Litmus workflows are `litmuschaos:k8s`, `litmuschaos:litmus-checker` etc.
+With ChaosCenter, users get the privilege to use their own/custom image registries for Chaos Workflows.
+
+## Update Chaos Workflow Image Registry
+To updated the Chaos Workflow Image Registry, user can go to Settings in ChaosCenter. In settings, there will be tab named Image Registry. On clicking the Image Regitry tab, user can see that the default Registry server is `docker.io`, Registry name is `litmuschaos` and it is a Public registry.
```
Comment (Saranya-jena) [inline website/docs/user-guides/image-registry.md]: Regitry -> Registry

**Comment by amityt** [comment]
Have made the requested changes! Thanks for the review @Saranya-jena @vanshBhatia-A4k9

**Review by Saranya-jena — APPROVED** [review]
Awesome thanks! 🚀

File: website/docs/user-guides/image-registry.md [hunk 6]
```diff
@@ -3,5 +3,54 @@ id: image-registry
...
 sidebar_label: Using different Image Registries
 ---
+---
+
+## Image Registry
+A container image registry can be defined as a collection of repositories that store container image.
+These can be either public or private. Few of the container image registries are Docker, Red Hat Quay, Google Container Registry. 
+By default LitmusChaos uses DockerHub for managing the different images. These images are then used in Chaos Workflows. Few images that are used in the Litmus workflows are `litmuschaos:k8s`, `litmuschaos:litmus-checker` etc.
+With ChaosCenter, you get the privilege to use your own/custom image registries for Chaos Workflows.
+
```
Comment (S-ayanide) [inline website/docs/user-guides/image-registry.md:12]: Can you please add a `Before you begin` section where the associated concepts that are required as a pre-requisite will be referenced!

File: website/docs/user-guides/image-registry.md [hunk 7]
```diff
@@ -3,5 +3,54 @@ id: image-registry
...
+          - -operation=apply
+          - -timeout=400
+          - -app=bank-of-anthos
+          - -scope=cluster
+    - name: install-chaos-experiments
+      container:
+        image: docker.io/testing-reg/k8s:latest
+```
+
+
```
Comment (S-ayanide) [inline website/docs/user-guides/image-registry.md:60]: Can you please add some associated `Learn More` links!

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Review by S-ayanide — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

File: website/docs/concepts/chaoshub.md [hunk 8]
```diff
@@ -4,4 +4,88 @@ title: ChaosHub
 sidebar_label: ChaosHub
 ---
 
-## Coming soon
+---
+
+## Introduction
```
Comment (arkajyotiMukherjee) [inline website/docs/concepts/chaoshub.md]: Intro may not have a heading called intro...you can just start with the intro paragraph

File: website/docs/user-guides/image-registry.md [hunk 9]
```diff
@@ -3,5 +3,61 @@ id: image-registry
 title: Using different Image Registries in a Workflow
 sidebar_label: Using different Image Registries
 ---
+---
+
+## Image Registry
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/image-registry.md]: This heading might not be required

Comment (arkajyotiMukherjee) [inline website/docs/user-guides/image-registry.md]: This heading might not be required

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**